scripts/create_tables.sql

```diff
@@ -1 +1,31 @@
 -- create doctors, patients, and appointments tables
+-- family_name,given_name,gender,born_on,specialty -->
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on TEXT,
+  speciality TEXT
```

Comment:
Careful here, you made the column speciality, then refer to it later as specialty.